hydra/hydra.go

```diff
@@ -129,9 +129,9 @@ func NewFluentRecordLTSV(key string, line []byte) *fluent.TinyFluentRecord {
 }
 
 func NewFluentRecordJSON(key string, line []byte) *fluent.TinyFluentRecord {
-	data := make(map[string]interface{})
-	err := json.Unmarshal(line, &data)
+	data, err := djson.DecodeObject(line)
 	if err != nil {
+		data = make(map[string]interface{}, 1)
```

Comment:
Thanks for a suggestion.
I compared with AllocString() and djson.DecodeObject().
CPU usage is about 30% lower when using djson.DecodeObject().